control-plane/tests/test_service_restore_validation.py

```
@@ -0,0 +1,1315 @@
+                rf"lifecycle transition chain for case record '{promoted_case.case_id}' "
+                r"contains no-op transition: "
+                r".* previous_lifecycle_state 'open' matches lifecycle_state 'open'"
+            ),
+        ):
+            restored_service.restore_authoritative_record_chain_backup(backup)
+        self._assert_authoritative_store_empty(restored_store)
+
+    def test_service_phase21_restore_rejects_unsupported_transition_subject_family(
+        self,
+    ) -> None:
+        _store, service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.12)

[warning] Unpacked variable `promoted_case` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)